orchagent/muxorch.cpp

```diff
@@ -1168,7 +1154,7 @@ void MuxOrch::updateRoute(const IpPrefix &pfx, bool add)
                         pfx.to_string().c_str(), nexthop.to_string().c_str());
                 continue;
             }
-            SWSS_LOG_INFO("setting route %s with nexthop %s %" PRIx64 "",
+            SWSS_LOG_NOTICE("setting route %s with nexthop %s %" PRIx64 "",
                 pfx.to_string().c_str(), nexthop.to_string().c_str(), next_hop_id);
             mux_multi_active_nh_table[pfx] = nexthop;
```

Comment:
Is this still being used?

Reply:
It's not necessary, but it can help us tell when this update is being triggered. We can still skip if the route is pointing to a single nexthop